### PR TITLE
docs(monolithic-engine): add docs

### PR DESCRIPTION
**Pull request by gergelyke**
Preview: https://deploy-preview-20--styletron.netlify.app/

### CODE REVIEW

**Comment by tajo** [comment]
I wonder if we should rather integrate this documentation into different sections so atomic and monolithic are treaded equally. Now, everything assumes atomic (like the pitfalls sections or getting started).

So there could be still a page discussing atomic vs monolithic but whenever it's important (setup, concepts...) it would mention both engines.

Unless it's not our strategy to treat them as equally viable options (fine with me too).

**Comment by gergelyke** [comment]
that makes sense - I'll update this PR next week

**Comment by gergelyke** [comment]
@tajo just updated this PR with your recommendation

File: pages/getting-started.mdx [hunk 1]
```diff
@@ -12,15 +12,36 @@ yarn add styletron-engine-atomic styletron-react
...
+
+### [styletron-engine-monolithic](https://www.npmjs.com/package/styletron-engine-monolithic)
+
+This package is an alternative implementation of the [styletron-standard](https://www.npmjs.com/package/styletron-standard) engine interface. The monolithic engine is a superset of the atomic engine - everything that works in the atomic engine will keep working with the monolithic engine. In comparison with the atomic engine:
+- class names are stable (hashes) in the monolithic engine, so components can be cross-referenced,
+- using the monolithic engine, you can mix longhand and shorthand properties,
+- media queries are not sorted, since the order is always respected,
+- it supports [@supports](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports).
+
+These improvements are made possible by sacrificing the minimal CSS footprint the atomic engine thrives for.
```
Comment (sandgraham) [inline pages/getting-started.mdx:31]:
```suggestion
These improvements are made possible by sacrificing the minimal CSS footprint the atomic engine strives for.
```

**Comment by gergelyke** [comment]
> Anyone else think the home page is a bit confusing nowadays? The copy talks about component oriented styling but the example is `useStyletron`. It also assumes that the atomic engine is still the default Styletron client.

how would you change the copy there?

**Comment by sandgraham** [comment]
@gergelyke The text describing the first example still references the original component (`styled`) example. So I think we should update the copy or just restore the original example. This could probably be done separately though, I don't think it is directly related to your changes.

**Comment by gergelyke** [comment]
@sandgraham I'll send a new PR addressing that

**Pull request merged**